### DISCUSSION TITLE
[Bug]: can't install asset-metadata-class-definitions

### DISCUSSION THREAD

**Discussion by gitmini** [pimcore/pimcore#16842] [Q&A / Support]
### Pimcore version

v11.2.1

### Steps to reproduce

execute composer require pimcore/asset-metadata-class-definitions, errors:
Composer could not detect the root package (pimcore/demo) version, defaulting to '1.0.0'. See https://getcomposer.org/root-version

In PackageDiscoveryTrait.php line 376:
                                                                                                                                                                                          
  Could not find a matching version of package pimcore/asset-metadata-class-definitions. Check the package spelling, your version constraint and that the package is available in a stab  
  ility which matches your minimum-stability (dev). 

### Actual Behavior

composer require pimcore/asset-metadata-class-definitions

### Expected Behavior

Composer could not detect the root package (pimcore/demo) version, defaulting to '1.0.0'. See https://getcomposer.org/root-version

In PackageDiscoveryTrait.php line 376:
                                                                                                                                                                                          
  Could not find a matching version of package pimcore/asset-metadata-class-definitions. Check the package spelling, your version constraint and that the package is available in a stab  
  ility which matches your minimum-stability (dev).

**Comment by kingjia90** [comment]
Asset metadata class definitions is an Enterprise bundle, it requires the private repository to be appended.
By looking at the pimcore/demo issue, probably there are some other issues, but please double-check [here](https://github.com/pimcore/demo/blob/5d8942827a93266a0c4b9e6a6dd90812af9d790e/composer.json#L62-L70) 
```
  "repositories": {
    "dev": {
      "type": "path",
      "url": "./dev/*/*",
      "options": {
        "symlink": true
      }
    },
    "enterprise": {
      "type": "composer",
      "url": "https://enterprise.repo.pimcore.com/"
    }
  }
```
for pimcore/demo, not sure, but please match the version on the ones available here https://github.com/pimcore/demo/releases
